ChangeLog

```diff
@@ -1,3 +1,54 @@
+3.26.0:
+	- Fixed issue where rhel >8 packages would not have correct openssl
+	  dependency version (ENT-12587)
+	- Added http_port and getattr selinux permissions as needed for selinux
+	  policy on rhel-8 and rhel-9 (ENT-12954)
+	- Fixed `cf-net connect` to return exit code 1 in case of error (CFE-4414)
```

Comment:
I have a vague memory of these backticks causing issues somewhere. But I don't remember what it was.
```suggestion
	- Fixed 'cf-net connect' to return exit code 1 in case of error (CFE-4414)
```

Reply:
yeah, we need a better way to automate picking up recent changes I guess... we fixed that in the generator but I made these manually.

```
            # ENT-7979: we don't want to see backticks ` in our changelogs
            line = line.decode().rstrip('\r\n').replace('`',"'")
```